iOS/Core/Pixel.swift

```diff
@@ -204,6 +204,11 @@ public class Pixel {
         static let phone = "phone"
     }
 
+    public enum TargetType: String {
```

Comment:
`TargetType` sounds kinda generic.  Would you be open to rename this to something more specific, like `SourceProcess`? (this would describe apps and extensions fairly well IMO).